stable/karma/OWNERS

```diff
@@ -1,4 +1,6 @@
 approvers:
 - davidkarlsen
+- prymitive
```

Comment:
I think you need to be in the maintainer section of the Chart.yaml too - see the README

Reply:
and also chart version needs bump to pass lint etc

Reply:
Yeah, thanks. I missed that part and went straight for `OWNERS`, plus chart version needs a bump too